projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts

```diff
@@ -1,14 +1,19 @@
+import * as authentication from './auth-forms';
+
 const nextGroupButtonSelector =
   'cx-config-previous-next-buttons div div:last button';
 const previousGroupButtonSelector =
   'cx-config-previous-next-buttons div div:first button';
 const addToCartButtonSelector = 'cx-config-add-to-cart-button button';
 
-export function clickOnConfigureButton() {
+const email = 'cpq02@sap.com';
```

Comment:
We should rather take users that are available in standard SPA test data, cpq02@sap.com stems from an internal extension